test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/ExportTests.cs

```diff
@@ -30,7 +30,6 @@ public ExportTests(HttpIntegrationTestFixture fixture)
         }
 
         [Theory]
```

Comment:
nit: This can be change to `Fact`. And the test can also be renamed to be more specific (maybe something like WhenRequestingExportForResourceWithId)